RunGcm-filters-leticia.ipynb

```diff
@@ -0,0 +1,3088 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ssh_SmallScale = ssh - ssh_filtered
</pre><p>Again, could add another line here just typing</p><pre class="ql-syntax" spellcheck="false">ssh_SmallScale  # this is a dask array again
</pre><p>to show people that this is still a dask array</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>